Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -160,14 +160,12 @@ - (void) UnlockPlanes {
  Returns the orientation of the app based on the current status bar orientation.
 */
 - (UIInterfaceOrientation)orientation {
-    auto sharedApplication = [UIApplication sharedApplication];
-    auto window = sharedApplication.windows.firstObject;
-    if (@available(iOS 13.0, *)) {
-        return window.windowScene.interfaceOrientation;
-    }
-    else {
-        return [[UIApplication sharedApplication] statusBarOrientation];
-    }
+    UIApplication* sharedApplication = [UIApplication sharedApplication];
+#if (__IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_13_0)
+    return [[[[sharedApplication windows] firstObject] windowScene] interfaceOrientation];
+#else
+    return [sharedApplication statusBarOrientation];
```

Comment:
Should we keep the runtime @available check in the else case since we might still be running on an iOS version >= to 13?

Reply:
Previously, we required at least the iOS 13 SDK to compile, but at runtime it would work on iOS 13 (using `window.windowScene.interfaceOrientation`) or iOS <13 (using `[[UIApplication sharedApplication] statusBarOrientation]`). Is that right?
Is this change trying to make it so you can compile the code against an older SDK that doesn't know anything about `sharedApplication.windows.firstObject` or `window.windowScene.interfaceOrientation`?

Reply:
I changed the @available for a conditional compilation because on iOS 13.0, it didn't compile: `'statusBarOrientation' was deprecated in iOS 13.0` error. (I would have liked keep `@available` which was more elegant :p)

Reply:
Im afraid that `if(!@available(..))` is not allowed. I suggest you this :

 ```
   UIApplication* sharedApplication = [UIApplication sharedApplication];
#if (__IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_13_0)
    return [[[[sharedApplication windows] firstObject] windowScene] interfaceOrientation];
#else
    if (@available(iOS 13.0, *)) {
        return [[[[sharedApplication windows] firstObject] windowScene] interfaceOrientation];
    }
    else {
        return [sharedApplication statusBarOrientation];
    }
#endif

```

Reply:
Ok I think that is reasonable. I was trying to avoid duplicating that line, but I think my logic was actually wrong anyway. Thanks for making this contribution!

Reply:
Yeap, I would also have liked to avoid duplicating these lines :|